src/main/java/seedu/saveit/model/UniqueIssueList.java

```diff
@@ -69,6 +69,7 @@ public void setIssue(Issue target, Issue editedIssue) {
         }
 
         internalList.set(index, editedIssue);
+        int i = 1;
```

Comment:
a meaningful variable name will be more appropriate

Reply:
Now removed